src/OrchardCore.Modules/OrchardCore.ContentTypes/Deployment/ContentDefinitionDeploymentStep.cs

```diff
@@ -3,13 +3,17 @@
 namespace OrchardCore.ContentTypes.Deployment
 {
     /// <summary>
-    /// Adds all content definitions to a <see cref="DeploymentPlanResult"/>. 
+    /// Adds selected content definitions to a <see cref="DeploymentPlanResult"/>. 
     /// </summary>
     public class ContentDefinitionDeploymentStep : DeploymentStep
     {
         public ContentDefinitionDeploymentStep()
         {
-            Name = "ContentDefinition";
+            Name = "SelectedContentDefinition";
```

Comment:
Why change the name?

Reply:
I didn't change the name.  I reuse the same filename name for this new step.  Because content steps are also separated in two different steps ("all" and "selected"), so I thought it was the trend.